backend/services/PostTypeHandler.py

```
@@ -0,0 +1,42 @@
+#all helper functions related to post, will be here
+from sqlalchemy.orm import Session
+from models.post import Post, PostMedia, PostDocument, Event
+from sqlalchemy.orm import Session
```

Ruff (0.8.2)

Redefinition of unused `Session` from line 2

Remove definition: `Session`

(F811)